mono/utils/mono-threads-coop.c

```diff
@@ -273,13 +320,13 @@ mono_threads_core_end_global_suspend (void)
 void*
 mono_threads_enter_gc_unsafe_region (void)
 {
-	return mono_threads_reset_blocking_start ();
+	return mono_threads_reset_blocking_start (0);
 }
 
 void
 mono_threads_exit_gc_unsafe_region (void *regions_cookie)
 {
-	mono_threads_reset_blocking_end (regions_cookie);
+	mono_threads_reset_blocking_end (regions_cookie, 0);
```

Comment:
This is wrong is a few ways.

Passing zero when the argument is a pointer.

Further more, passing zero will cause a segfault in the code stack saving code when it tries to load it.

Finally, the stack address start should be weaved into this one too.